org.apache.netbeans.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id": "org.apache.netbeans",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
```

Comment:
Consider switching to openjdk11 here to get important SSL fixes.

Reply:
Apache Netbeans officially supports Java 9 and 10, not 11.

Reply:
Is there a new release planned soon? Java 9 and 10 are already EOL.

Reply:
As far as I know they're working on release 10, proposed release date is November:
https://cwiki.apache.org/confluence/display/NETBEANS/Apache+NetBeans+Release+Roadmap